### ISSUE TITLE
Pop up on map page re finding own location

### ISSUE THREAD

**Issue by RichardFoxBC** [NERC-CEH/irecord-butterflies-app#41]
Could we have a pop up tip message on the map page please for the first time that someone uses it please? This should point to the button in top right corner of the map and say something like "Tapping this button will centre the map on your current location. This may take a little time."

You should be able to then hide the pop-up if you wish to so that it doesn't appear every time.

**Comment by kazlauskis** [comment]
Thanks, I wonder if it is not better to improve the interface to make it clearer what the button does instead. Which one of these icons resembles GPS/current-location to you the most?

**Generic:**
<img width="98" alt="Screenshot 2021-04-05 at 11 59 45" src="https://user-images.githubusercontent.com/7383162/113567473-98a74000-9606-11eb-8fa8-8f24367b2db4.png">

**iOS:**
<img width="120" alt="Screenshot 2021-04-05 at 11 59 57" src="https://user-images.githubusercontent.com/7383162/113567478-99d86d00-9606-11eb-994b-18ea28cf7197.png">

**Android:**
<img width="109" alt="Screenshot 2021-04-05 at 12 00 08" src="https://user-images.githubusercontent.com/7383162/113567481-9a710380-9606-11eb-8fa2-c24cf11b8114.png">

**Comment by RichardFoxBC** [comment]
I'd probably go with the Android option of the three above (despite being an iOS user), but I still think that a pop-up message (that can be dismissed so it doesn't reappear) would be a good idea too

**Comment by RichardFoxBC** [comment]
We didn't talk about this during the meeting this morning and no-one else has responded to Karolis' icon options. I still think this pop up message would be a good idea, ideally for the first release of the app

**Comment by DavidRoy** [comment]
I agree on the 'android' icon and ideally a 'first-time' pop-up message.

**Comment by RichardFoxBC** [comment]
As if to prove the need for this popup, I had this message just now from a tester: 
...  My big issue is with the method of recording the location - it didn’t seem to default to using my current location, so for every sighting I was presented with a full UK map and had to zoom in to my little corner of the north of Scotland. After 4 (fictional) sightings in different locations I was bored!  At one point I tapped the symbol top right of the map, not knowing what it was going to do, and it proceeded to show the twirling circle thing for ages. I’ve no idea what it was doing, but I eventually ignored it and tapped to go back to the previous page.
